spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:each) do
+      get '/users/0/posts'
```

Comment:
2. I think it would be a good idea to count from 1 instead of zero, right? Because all websites should present valid numbers. Or, maybe some users don't have learned to count from zero.